doc/environment_variables.md

```diff
@@ -0,0 +1,61 @@
+gRPC environment variables
+--------------------------
+
+gRPC C core based implementations (those contained in this repository) expose
+some configuration as environment variables that can be set.
+
+* GRPC_ABORT_ON_LEAKS
+  A debugging aid to cause a call to abort() when gRPC objects are leaked past
+  grpc_shutdown()
+
+* GRPC_GOOGLE_CREDENTIALS_ENV_VAR
+  The path to find the credentials to use when Google credentials are created
+
+* GRPC_SSL_CIPHER_SUITES
+  A colon separated list of cipher suites to use with OpenSSL
+
+* GRPC_POLL_STRATEGY [posix-style environments only]
+  Declares which polling engines to try when starting gRPC.
+  This is a comma-separated list of engines, which are tried in priority order
+  first -> last.
+  Available polling engines include:
+  - epoll (linux-only) - a polling engine based around the epoll family of
+    system calls
+  - poll - a portable polling engine based around poll(), intended to be a
+    fallback engine when nothing better exists
+  - legacy - the (deprecated) original polling engine for gRPC
+
+* GRPC_TRACE
+  A comma separated list of tracers that provide additional insight into how
+  gRPC C core is processing requests via debug logs. Available tracers include:
```

Comment:
I wonder how to deal with updates to this list. Perhaps have a special tag inside a comment (I believe markdown uses html syntax for those) and a corresponding sanity script to remind ourselves to update this list if new tracers are added? Borderline overengineering though.